crates/ruma/examples/hello_isahc.rs

```diff
@@ -24,9 +23,9 @@ async fn hello_world(
     let room_id = client.send_request(get_alias::v3::Request::new(room_alias)).await?.room_id;
     client.send_request(join_room_by_id::v3::Request::new(&room_id)).await?;
     client
-        .send_request(send_message_event::Request::new(
+        .send_request(send_message_event::v3::Request::new(
             &room_id,
-            "1",
+            "1".into(),
```

Comment:
I think we should use `TransactionId::new()` here (which will require adding `rand` to the required features for the example, but that's okay).